examples/src/bin/tlsclient-mio.rs

```diff
@@ -243,20 +243,17 @@ struct Args {
     arg_hostname: String,
 }
 
-// TODO: um, well, it turns out that openssl s_client/s_server
-// that we use for testing doesn't do ipv6.  So we can't actually
-// test ipv6 and hence kill this.
-fn lookup_ipv4(host: &str, port: u16) -> SocketAddr {
+fn lookup_ip(host: &str, port: u16) -> SocketAddr {
```

Comment:
i think we could just delete this, as `TcpStream::connect` can just be passed `(args.arg_hostname.as_str(), port)` instead, and will connect to all the possible addresses that resolve from that: https://doc.rust-lang.org/std/net/trait.ToSocketAddrs.html